src/db/init_database.js

```diff
@@ -45,6 +51,20 @@ export async function populateTools() {
     documentationUrl: 'http://expressjs.com/en/4x/api.html',
     logoLargeUrl: 'https://camo.githubusercontent.com/fc61dcbdb7a6e49d3adecc12194b24ab20dfa25b/68747470733a2f2f692e636c6f756475702e636f6d2f7a6659366c4c376546612d3330303078333030302e706e67',
   });
+  await sequelize.Tool.create({
+    name: 'Mocha',
+    toolType: sequelize.ToolType.TEST,
+    websiteUrl: 'https://mochajs.org/',
+    documentationUrl: 'https://mochajs.org/#getting-started',
+    logoLargeUrl: 'https://worldvectorlogo.com/logo/mocha-1'
+  });
+  await sequelize.Tool.create({
+    name: 'Sequelize',
+    toolType: sequelize.ToolType.TEST,
```

Comment:
I think you said you were going to fix the toolType for this one, but I'm adding a comment to remind you.